tests/integration/local/invoke/test_integrations_cli_cdk.py

```diff
@@ -202,3 +181,37 @@ def test_invoke_with_parameters_overrides(self):
         self.assertIsNone(environ.get("TimeOut"))
         self.assertEqual(environ["MyRuntimeVersion"], "v0")
         self.assertEqual(environ["EmptyDefaultParameter"], "")
+
+    @pytest.mark.flaky(reruns=3)
+    def test_invoke_returns_expected_results_python_function_construct(self):
+        command_list = self.get_command_list(
+            "AwsLambdaFunctionStack/python-function-construct", event_path=self.event_path
+        )
+
+        process = Popen(command_list, stdout=PIPE, cwd=self.working_dir)
```

Comment:
We have a `run_command` util function defined [here](https://github.com/aws/aws-sam-cli/blob/b0dc9dad779ab0bc63c5688399429ee947b0c86f/tests/testing_utils.py#L26-L37). Can we all use the that one rather repeating over and over again?